idea/src/main/kotlin/io/paddle/idea/execution/state/PythonScriptCommandLineStateProvider.kt

```diff
@@ -20,6 +20,7 @@ class PythonScriptCommandLineStateProvider : PaddleTaskRunProfileStateProvider<R
 
         val additionalArgsLine = (context.originalRunConfiguration as PaddleRunConfiguration).commandLine.tasksAndArguments.toList()
         val additionalArgs = Paddle.parseCliOptions(additionalArgsLine)["extraArgs"]?.trim('"', '\'') ?: ""
+        val env = context.originalRunConfiguration.settings.env + mapOf("PYTHONPATH" to task.project.environment.pythonPath)
```

Comment:
hehe, what if `PYTHONPATH` is also specified by the user? I'd better merge it with the key (add to the beginning of path, separated by semicolon). This could help in cases when Paddle fails, I guess.